charts/kubernikus-system/charts/prometheus/templates/_prometheus.yaml.tpl

```diff
@@ -251,7 +251,9 @@ scrape_configs:
       target_label: __metrics_path__
       replacement: /api/v1/nodes/${1}:4194/proxy/metrics
 
-- job_name: 'blackbox-ingress'
+{{- range .Values.prometheus.region_probed_from }}
```

Comment:
My personal preference is to stick this config into the `extra_scrap_config` of the values. We'd have to dumb it down a bit though. Maybe even into a single global config for all installations. Like always probe from `proper.eu-nl-1.cloud.sap` and `proper.na-us-1.cloud.sap` and that's it.